torch/testing/_internal/optests/generate_tests.py

```diff
@@ -147,6 +147,34 @@ def new_method(*args, **kwargs):
                 result = method(*args, **kwargs)
             return result
 
+        if pytestmark := new_method.__dict__.get("pytestmark"):
+            import pytest
```

Comment:
Can you include a comment somewhere on how to actually mark a test with opcheck_only_one?